### DISCUSSION TITLE
令人振奋的消息来了！我们的网站焕然一新了！www.lantern.io

### DISCUSSION THREAD

**Discussion by MaxMKMax** [getlantern/lantern-forum-cn#532] [Announcements]
![Website Announcement China](https://github.com/getlantern/lantern-forum-cn/assets/155475142/9186dfaa-334a-44cb-9b0e-7546c11a805e)

请您查看全新的网站设计并关注更多的信息更新: www.lantern.io

**Comment by sobai14150** [comment]
你好  蓝灯激活如何购买 之前购买激活的页面失效无法打开
